### PR TITLE
Music: remove progression file support

### PR DESCRIPTION
**Pull request by breville**
Before Music Lab was integrated into our level system, it was possible to move through a progression provided by a JSON file.  Now that we are using our level system, this support has been removed.  

I've left our default progression file intact [here](https://github.com/code-dot-org/code-dot-org/blob/f5836a6cf533f0f1542ba46eccac07e244247538/apps/static/music/music-progression.json), since it serves as a nice reference of what can go into level definitions.

### CODE REVIEW

File: apps/src/music/views/MusicView.jsx [hunk 1]
```diff
@@ -306,19 +306,12 @@ class UnconnectedMusicView extends React.Component {
...
-  // Returns whether we have a progression file.
-  hasProgressionFile = () => {
-    return AppConfig.getValue('load-progression') === 'true';
-  };
-
   // Returns whether we have a progression.
   // Note that even a standalone level has a progression in the sense that we
   // will show instructions and feedback, but we'll only show them for that
   // standalone level.
   hasProgression = () => {
```
Comment (molly-moen) [inline apps/src/music/views/MusicView.jsx:312]: is it just `/projectbeats` that returns false for this method?

**Review by molly-moen — APPROVED** [review]
🎉 yay cleanup!

File: apps/src/music/views/MusicView.jsx (hunk 1 above)
Comment (breville) [reply apps/src/music/views/MusicView.jsx:312]: Good point, yes.

File: apps/src/music/utils/Loader.ts [hunk 2]
```diff
@@ -108,12 +86,6 @@ export const loadProgressionStepFromSource = async (
     );
     progressionStep = response.value.level_data;
     levelCount = 1;
```
Comment (sanchitmalhotra126) [inline apps/src/music/utils/Loader.ts]: Can the above logic also be reduced now as the only difference is levelCount = 1 if levelSource === LevelSource.LEVEL?

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice!

File: apps/src/music/views/MusicView.jsx [hunk 3]
```diff
@@ -296,33 +296,22 @@ class UnconnectedMusicView extends React.Component {
     logError(error);
   };
 
-  // Returns whether we just have a standalone level.
-  isStandaloneLevel = () => {
+  // Returns whether we just have a single level.
+  isSingleLevel = () => {
```
Comment (sanchitmalhotra126) [inline apps/src/music/views/MusicView.jsx:299]: great cleanup!

File: apps/src/music/utils/Loader.ts (hunk 2 above)
Comment (breville) [reply apps/src/music/utils/Loader.ts]: Good call; this led to a bunch more tidying up.

**Review by molly-moen — APPROVED** [review]

**Review by sanchitmalhotra126 — APPROVED** [review]

**Pull request merged**